src/quadpype/hosts/tvpaint/plugins/create/create_render.py

```diff
@@ -1210,6 +1210,11 @@ def get_instance_attr_defs(self):
                 label="Apply BG Color (as defined in settings)",
                 default=self.apply_background
             ),
+            BoolDef(
```

Comment:
I think it would be cleaner to merge `apply_background` and `apply_alpha_background` together in an enum option, configurable in the settings, this would change `apply_background`.

In the UI this should be a dropdown option menu named `Background` with three options:
- `Unchanged` or `No Operation` or `------------`
- `Flat color (Settings defined)`
- `Transparent`

Additional: We could imagine that the options of this enumerator are set in the settings, so we could define multiple project options (black, grey, transparent, ...).

Because having two attributes conflicting each other doesn't seems like a hood idea, and adding too much attributes in the UI / Settings for a similar purpose (editing the background) isn't a good idea ether.